spec/oas3.patched/openapi/public/components/schemas/Plan.yaml.rej

```diff
@@ -0,0 +1,14 @@
+--- spec/oas3.patched/openapi/public/components/schemas/Plan.yaml
```

Comment:
This file is created because the plan memory patch was upstreamed.  We need to remove the upstreamed patch before merging.